stix_shifter_modules/msatp/configuration/lang_en.json

```diff
@@ -20,6 +20,24 @@
         "sni": {
             "label": "Server Name Indicator",
             "description": "The Server Name Indicator (SNI) enables a separate hostname to be provided for SSL authentication"
+        },
+        "options": {
+            "includeAlerts": {
+                "label": "Join events with Alerts?",
```

Comment:
Change `Alerts` with `alerts`. We are trying to keep labels sentence-case.

Reply:
done